interpreter/src/Language/Haskell/Interpreter.hs

```diff
@@ -0,0 +1,114 @@
+{-# LANGUAGE DeriveAnyClass      #-}
+{-# LANGUAGE DeriveGeneric       #-}
+{-# LANGUAGE DerivingStrategies  #-}
+{-# LANGUAGE FlexibleContexts    #-}
+{-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE RecordWildCards     #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+module Language.Haskell.Interpreter (runghc, CompilationError(..)) where
```

Comment:
Seems like there's some overlap with `Interpreter` in `plutus-playground`, but perhaps not worth consolidating.

Reply:
I'm silly and I hadn't realised the whole point of this is to share things.